test/frau-superagent-xsrf-token.spec.js

```diff
@@ -29,11 +29,11 @@ describe('frau-superagent-xsrf-token', function() {
 			.get('/api')
 			.reply(200);
 
-		var req = request
+		request
 			.get('http://localhost/api')
 			.use(xsrf)
-			.end(function() {
-				expect(req.req._headers).not.to.have.property('x-csrf-token');
+			.then(function(res) {
+				expect(res.request.req.headers).not.to.have.property('x-csrf-token');
```

Comment:
`_headers` was removed in Node 12:
https://nodejs.org/api/deprecations.html#DEP0066